xchange-coindeal/src/main/resources/logback.xml

```diff
@@ -0,0 +1,23 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
this has to potential to override logging settings in every applicaton that uses logback and this library
remove it

Reply:
Only this line right?

Reply:
no, the whole file.